cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/PartitionedFileSetArguments.java

```diff
@@ -170,4 +172,15 @@ public static Comparable convertFieldValue(String where, String kind, String fie
                       where, kind, stringValue, fieldName, fieldType.name()), e);
     }
   }
+
+  public static void addPartitions(Map<String, String> arguments, Iterator<Partition> partitionIterator) {
```

Comment:
This PR isn't implementing multiple PartitionFilters (which would be used for setting PartitionKeys).
Instead, this method is just a wrapper around `FileSetArguments#addInputPath`, for the use case of PartitionedFileSetDataset delegating to FileSetDataset, when only the input paths are set in the arguments (an no PartitionFilter).